tests/error/modifier_errors/unexpected_ctxt_maneger.err

```diff
@@ -0,0 +1,8 @@
+Error: Unexpected context manager (at $FILE:6:9)
+  | 
+4 | @guppy
+5 | def test() -> None:
+6 |     with True:
+  |          ^^^^ Unexpected context manager in a context manager
```

Comment:
This reads a bit akwardly. Maybe something like "Invalid modifier, expected one of `dagger`, `control(...)`, or `power(...)`" would be clearer?